maps/aurora/aurora-8_derelict.dmm

```diff
@@ -4334,7 +4334,6 @@
 /turf/simulated/floor/tiled/white,
 /area/derelict/hallway/southeast)
 "nt" = (
-/obj/machinery/computer/scan_consolenew,
```

Comment:
Same here.

Reply:
This is actually a thing on the derelict that should stay.